### PR TITLE
Feature/305

### PR DESCRIPTION
**Pull request by GalochkinKirov**
https://github.com/OAGr/foretold/issues/305

Please, consider this PR after all previous.

### CODE REVIEW

**Comment by GalochkinKirov** [comment]
Screenshot_20190628_095232

Page example.

File: packages/client/src/pages/Measurable/MeasurableEdit.re [hunk 1]
```diff
@@ -25,23 +25,26 @@ module WithEditMutation = {
...
     let date = showDescriptionDate == "TRUE" ? labelOnDate : "";
     let m =
       GraphQL.make(
         ~id,
         ~input={
           "name": name,
-          "labelCustom": labelCustom |> E.O.some,
-          "labelProperty": labelProperty |> E.O.some,
-          "labelOnDate": date |> Js.Json.string |> E.O.some,
+          "labelCustom": labelCustom |> Rationale.Option.some,
```
Comment (OAGr) [inline packages/client/src/pages/Measurable/MeasurableEdit.re]: Why did you change this? I'm trying to stay away from the use of ``Rationale`` directly, unless we want to dramatically change use of the E library.

Comment (GalochkinKirov) [reply packages/client/src/pages/Measurable/MeasurableEdit.re]: Because it is not readable.
